src/Engine/ProtoAssociative/CodeGen.cs

```diff
@@ -453,7 +453,7 @@ private void UpdateType (string ident, int funcIndex, ProtoCore.Type dataType)
             ProtoCore.Type datatype, 
             int datasize = ProtoCore.DSASM.Constants.kPrimitiveSize, 
             bool isStatic = false,
-            ProtoCore.Compiler.AccessSpecifier access = ProtoCore.Compiler.AccessSpecifier.kPublic,
```

Comment:
Probably it is a good time to change ``AccessSpecifier`` to more accurate term ``AccessModifier`` :-)